src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -1428,7 +1420,7 @@ public void userAuthentication() throws Exception {
       // check password
       assertTrue(
           client.authenticateUser(creds, "root", s2pp(SharedMiniClusterBase.getRootPassword())));
-      assertFalse(client.authenticateUser(creds, "root", s2pp("")));
+      assertFalse(client.authenticateUser(creds, "otheruser", s2pp("")));
```

Comment:
This test was broken, but it's not clear why. This might actually require further investigation in Accumulo itself. It seemed to return true, regardless of the password, if the root user tried to authenticate themselves. A good test here might be to create a new user, and try to authenticate that new user, with their new password and a bad password, instead of trying to test with the root user, which is also the current user.